README.md

```diff
@@ -22,6 +22,10 @@ Go to https://excalidraw.com to start sketching.
 
 Read our [blog](https://blog.excalidraw.com) and follow the [guides](https://howto.excalidraw.com) to learn more about Excalidraw and how to use it effectively.
 
+## Libraries
```

Comment:
```suggestion
## Browse libraries
```

Reply:
I'd actually use something like `Asset libraries`, or `Shape libraries`, or similar.. for new users it may not be obvious what a library is in the first place.